tests/common.go

```diff
@@ -5444,3 +5444,66 @@ func WaitForPoolStatusToUpdate(nodeSelected node.Node, expectedStatus string) er
 	_, err := task.DoRetryWithTimeout(t, 10*time.Minute, 1*time.Minute)
 	return err
 }
+
+// DeleteGivenPoolInNode deletes pool with given ID in the given node
+func DeleteGivenPoolInNode(stNode node.Node, poolIDToDelete string) (err error) {
+
+	log.InfoD("Setting pools in maintenance on node %s", stNode.Name)
+	if err = Inst().V.EnterPoolMaintenance(stNode); err != nil {
+		return err
+	}
+	//Waiting for cli to work
+	time.Sleep(2 * time.Minute)
+
+	status, err := Inst().V.GetNodeStatus(stNode)
+	if err != nil {
+		return err
+	}
+	log.InfoD("Node [%s] has status: [%v] after entering pool maintenance", stNode.Name, status)
+
+	expectedStatus := "In Maintenance"
+	if err = WaitForPoolStatusToUpdate(stNode, expectedStatus); err != nil {
+		return fmt.Errorf("node %s pools are not in status %s. Error:%v", stNode.Name, expectedStatus, err)
```

Comment:
NIT , Instead of "Error" can we keep it to "Err" to keep uniformity ?

Reply:
fixed